x/xvalidator/validator.go

```diff
@@ -13,8 +14,11 @@ const (
 	maxPort = 65535
 
 	portSeparator = ":"
+	envSeparator  = "="
 )
 
+var envNameRegexp = regexp.MustCompile("^[a-zA-Z][a-zA-Z0-9_]*$")
```

Comment:
we could enforce capital and _ only

Reply:
numbers are ok

Reply:
> we could enforce capital and _ only

this is valid regex for env variable:
- numbers are allowed (but not at 1st pos)
- all letter cases are allowed - so some people use lower and some upper cases and all of them are valid

I don't see the value of changing standard env naming. If there is some we can consider it, otherwise let's keep to what devs used to.